vault-core/src/main/java/org/apache/jackrabbit/vault/packaging/registry/PackageTaskBuilder.java

```diff
@@ -37,9 +37,17 @@
     /**
      * Sets the type of this task
      * @param type the type
-     * @return this.
+     * @return the parent execution plan builder.
```

Comment:
@tripod Was that really supposed to return the `ExecutionPlanBuilder` instead of `PackageTaskBuilder`?